### PR TITLE
[Function][Admin] Create Management Words Function

### PR DESCRIPTION
**Pull request by kaz-m-93**
i created word management function(edit, destroy)
asana:
https://app.asana.com/0/1199094899705013/1199094899705091

### CODE REVIEW

**Review by CharlotteDS — CHANGES_REQUESTED** [review]
Please update your `radio_button` into a `check_box` so that we can have a cleaner and simpler implementation to get `true` or `false` value for `is_correct`.

**Comment by kaz-m-93** [comment]
i changed radio button to check box

File: app/views/admin/words/_form.html.erb [hunk 1]
```diff
@@ -12,8 +12,8 @@
             <%= f.fields_for :choices do |s| %>
                 <div class="form-group choice-form">
                     <%= s.text_field :content, class: "form-control choice-text" %>
-                    <!-- ↓ s.radio_button doesnt work, so its a solution for it -->
-                    <div class="choice-radio"><%= f.radio_button :correct_no, s.index, checked: s.object.is_correct ? true : false %></div>
+                    <!-- ↓ for simple implements, use check box instead of radio button -->
+                    <div class="choice-radio"><%= s.check_box :is_correct, value: true , checked: s.object.is_correct ? true : false %></div>
```
Comment (CharlotteDS) [inline app/views/admin/words/_form.html.erb:16]: No need to add `value: true , checked: s.object.is_correct ? true : false`.
The value will automatically be true or false, depending if it is checked.
In edit page, the check boxes will automatically get their values from the word's choices.

**Review by CharlotteDS — COMMENTED** [review]
I want you to add a custom validation in the `word.rb` to check that only one choice is checked.

**Review by CharlotteDS — APPROVED** [review]

**Pull request merged**